docs/reference/mapping/types/range.asciidoc

```diff
@@ -4,6 +4,13 @@
 <titleabbrev>Range</titleabbrev>
 ++++
 
+A field that represents a relation between a lower and upper bound, using the
+relations `gt`, `gte`, `lt` and `lte`. They can be used for querying, and have
+limited support for aggregations. The only supported aggregations are
+{ref}/search-aggregations-bucket-histogram-aggregation.html[histogram],
+{ref}/search-aggregations-bucket-range-aggregation.html[range] and
```

Comment:
Just tried `range` aggregation on simple `integer_range` and `date_range` fields on 7.7 and master, gettting errors for both that these fields are not supported for this type of field. Maybe I'm missing something, what did you check or refer to when adding this?